spec/services/conviction_decision_tree_spec.rb

```diff
@@ -129,8 +131,25 @@
   end
 
   context 'when the step is `motoring_endorsement`' do
-    let(:step_params) { { motoring_endorsement: 'anything' } }
-    it { is_expected.to have_destination(:known_date, :edit) }
+    let(:conviction_subtype) { :adult_penalty_notice }
+    let(:motoring_endorsement) {GenericYesNo::YES }
+    let(:step_params) { { motoring_endorsement:  motoring_endorsement } }
+
+    context 'when subtype equal adult_penalty_notice with a endorsement' do
+      let(:conviction_subtype) { :adult_penalty_notice }
```

Comment:
You don't need line 139, because you've got line 134 in this context.

But, for clarity, it might be worth using sub-contexts "when subtype equal adult_penalty_notice" and "when subtype equal adult_disqualification".

Another option is to move line 134 here, and copy it into the other context, too. 

Either way, specifying the conviction_subtype would be consistent for each context.